src/dynamic-plugin-sdk.ts

```diff
@@ -0,0 +1,603 @@
+/* eslint-disable */
+import * as React from 'react';
+import * as _ from 'lodash';
+
+const HOOK_POLL_DELAY = 2000; // change this if you want the useHook to go faster / slower on polls
```

Comment:
If for whatever reason 2 seconds is too long or too short, you can just modify this constant to change the poll rate of the hook.